docs/configuration/listeners/listeners.rst

```diff
@@ -9,17 +9,17 @@ Each individual listener configuration has the following format:
 .. code-block:: json
 
   {
-    "port": "...",
+    "address": "...",
     "filters": [],
     "ssl_context": "{...}",
     "bind_to_port": "...",
     "use_proxy_proto": "...",
     "use_original_dst": "..."
   }
 
-port
-  *(required, integer)* The TCP port that the listener should listen on. Currently only TCP
-  listeners are supported which bind to all local interfaces.
+address
+  *(required, string)* The address that the listener should listen on. Currently only TCP
+  listeners are supported.
```

Comment:
Can we get an example of what a TCP listener address looks like, and how to specify wildcard vs. specific.